### PR TITLE
Adding the ability to clear all types from store

### PR DESCRIPTION
**Pull request by williamsbdev**

### CODE REVIEW

**Comment by toranb** [comment]
Could we rename the method from clearAll() to clear() instead? It helps keep the overall API to just 5 methods in size :)

File: tests/unit/store-test.js [hunk 1]
```diff
@@ -325,6 +325,32 @@ test("clear will destroy everything for a given type", function() {
...
+  });
+
+  store.push("cat", {
+    id: 1,
+    color: "red"
+  });
+
+  store.clearAll();
+
+  equal(store.find("person").get("length"), 0);
```
Comment (toranb) [inline tests/unit/store-test.js:353]: would you mind adding a "pre clear" assertion to show you get 1 length before the clear? It helps tell the story (before and after) like :)

**Comment by williamsbdev** [comment]
I've made the recommended changes. Let me know if this is what you are looking for.

File: tests/unit/store-test.js [hunk 2]
```diff
@@ -325,6 +325,35 @@ test("clear will destroy everything for a given type", function() {
...
+
+  store.push("cat", {
+    id: 1,
+    color: "red"
+  });
+
+  equal(store.find("person").get("length"), 2);
+  equal(store.find("cat").get("length"), 1);
+
+  store.clear();
```
Comment (toranb) [inline tests/unit/store-test.js:351]: I love how this reads :) thanks for taking the time to polish this!

**Comment by toranb** [comment]
FYI - just did the npm publish so 0.6.0 is up!

**Pull request merged**